apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -103,20 +107,24 @@ export class BulkScriptAction extends BaseScriptAction {
       commandToRun: this._commandToRun,
       customParameterValues,
       isQuietMode: isQuietMode,
-      isIncrementalBuildAllowed: this.actionName === 'build',
+      isIncrementalBuildAllowed: this._isIncrementalBuildAllowed,
       ignoreMissingScript: this._ignoreMissingScript,
-      ignoreDependencyOrder: this._ignoreDependencyOrder
+      ignoreDependencyOrder: this._ignoreDependencyOrder,
+      packageDepsFilename: Utilities.getPackageDepsFilenameForCommand(this._commandToRun)
```

Comment:
In order for me to finish the pre and post npm script PR, this step will have to happen later, most likely in ProjectTask itself, to facilitate creating these files for preCommandToRun and postCommandToRun. But I can do that change after this PR is merged, if you like.

Reply:
That sounds good to me.